packages/editor/src/panels/properties/common/NodeEditor.tsx

```diff
@@ -73,6 +73,41 @@ class NodeEditorErrorBoundary extends React.Component<INodeErrorProps, INodeErro
   }
 }
 
+const documentationMap = {
+  GLTFComponent: 'ModelComponent',
+  RigidBodyComponent: 'RigidbodyComponent',
+  TriggerCallbackComponent: 'TriggerComponent'
+}
+
+const documentationList = [
+  'ModelComponent',
```

Comment:
I do wish we could somehow source this from a dynamic source of truth but I think it's fine for now.

Reply:
I agree. When I started I assumed the slugs would be available on the `component`s directly. Maybe we can add something to the build step for the docs